backend/src/modules/auth/middleware.ts

```diff
@@ -77,7 +77,7 @@ export const protectedRoute = (
 				if (!userRoles || !hasAllRoles(userRoles, roles)) {
 					// User does not have all the required roles.
 					return res.status(403).json({
-						sucess: false,
+						success: false,
```

Comment:
I think we will have to update this test too? from backend/tests/modules/auth/ lines 95-99:
```const app = createAppWithMiddleware(["role1", "role2"]);
		await request(app).get("/test").set("Authorization", "Bearer valid-token").expect(403, {
			**sucess**: false, // Note the typo from the middleware
			message: "User must have all of the following roles: [role1,role2]",
		});
```